control-plane/config/200-controller/200-addressable-resolver-cluster-role.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: kafka-addressable-resolver
+  labels:
+    kafka.eventing.knative.dev/release: devel
+    duck.knative.dev/addressable: "true"
+# Do not use this role directly. These rules will be added to the "addressable-resolver" role.
+rules:
+  - apiGroups:
+      - eventing.knative.dev
+    resources:
+      - kafkasinks
```

Comment:
should we split this file?

Reply:
There is already an addressable resolver for `KafkaSinks` in https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/control-plane/config/200-webhook/200-addressable-resolver-cluster-role.yaml (I'd change the location to be under `200-controller` rather than `200-webhook`)